modules/geo-layers/src/tile-layer/tile-layer.js

```diff
@@ -21,7 +21,9 @@ const defaultProps = {
   maxCacheSize: null,
   maxCacheByteSize: null,
   refinementStrategy: STRATEGY_DEFAULT,
-  zRange: null
+  zRange: null,
+  throttleRequests: true,
+  maxRequests: 8
```

Comment:
Thoughts on what the default should be? The `RequestScheduler` itself defaults to `6`.